### ISSUE TITLE
method definition, DEFAULT and OPTIONAL should give error

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1738]
```abap
  METHODS apps_list_installations
    IMPORTING
      outdated TYPE string OPTIONAL
      per_page TYPE i DEFAULT 30 OPTIONAL.
```
